workflow.rst

```diff
@@ -519,6 +521,44 @@ it via the marking::
     // contains the new value
     $marking->getContext();
 
+It is also possible to listen to these events by declaring event listeners
+with the following attributes:
+
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsAnnounceListener`
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsCompletedListener`
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsEnterListener`
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsEnteredListener`
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsGuardListener`
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsLeaveListener`
+* :class:`Symfony\\Component\\Workflow\\Attribute\\AsTransitionListener`
+
+These attributes do work like the
+:class:`Symfony\\Component\\EventDispatcher\\Attribute\\AsEventListener`
+attributes::
+
+    class ArticleWorfklowEventListener
+    {
+        public function __construct()
+        {
+        }
```

Comment:
Is it useful?

Reply:
Oh yes my bad, removed :slightly_smiling_face: